doc/guides/west/manifest.rst

```diff
@@ -942,12 +943,12 @@ An equivalent manifest in a single file would be:
        - name: mainline
          url-base: https://git.example.com/mainline
      projects:
-       - name: app
-       - name: lib
+       - name: app                  # imported
+       - name: lib                  # imported
          path: libraries/lib
-       - name: lib2
+       - name: lib2                 # imported
          path: libraries/lib2
-       - name: mainline
+       - name: mainline             # imported
```

Comment:
This one is declared directly, not imported

Reply:
Good catch, thx.